pyannote/audio/features/with_librosa.py

```diff
@@ -162,13 +161,13 @@ def get_features(self, y, sample_rate):
         data : (n_frames, n_mels) numpy array
             Features
         """
-
-        X = librosa.feature.melspectrogram(
+        mel_spec = librosa.feature.melspectrogram(
             y.squeeze(), sr=sample_rate, n_mels=self.n_mels,
-            n_fft=self.n_fft_, hop_length=self.hop_length_,
-            power=2.0)
+            n_fft=self.n_fft_, hop_length=self.hop_length_, power=2)
+
+        mel_spec = librosa.power_to_db(mel_spec, ref=np.max)
 
-        return librosa.amplitude_to_db(X, ref=1.0, amin=1e-5, top_db=80.0).T
```

Comment:
Can you explain why this is better?

Reply:
![amplitude_to_db](https://user-images.githubusercontent.com/11290637/61582569-5f5d3280-aafa-11e9-8870-e39f097600ff.png)
![power_to_db](https://user-images.githubusercontent.com/11290637/61582570-5f5d3280-aafa-11e9-89b8-a2194b0a6d03.png)

Not necessarily better. They seem equivalent in terms of quantity of information.